springfield/cms/models/pages.py

```diff
@@ -168,6 +168,8 @@ class WhatsNewPage(AbstractSpringfieldCMSPage):
     parent_page_types = ["cms.WhatsNewIndexPage"]
     subpage_types = []
 
+    ftl_files = ["firefox/whatsnew/evergreen"]
```

Comment:
There may be a risk here that we're speccing a single FTL file, but in doing so, lose the use of the ones in settings.DEFAULT_FLUENT_FILES. It might be that the WNP doesn't use any of the default strings (eg in the footer), but worth a check in case it does